Makefile

```diff
@@ -126,7 +126,7 @@ validate:
 ci: validate test release
 
 vendor:
-	export GO111MODULE=on \
+	GO111MODULE=on \
```

Comment:
You still need to `export`, otherwise `mod vendor` and `mod verify` won't have `GO111MODULE=on` set;

```console
$ GO111MODULE=on go env GO111MODULE && go env GO111MODULE && go env GO111MODULE
on


$ export GO111MODULE=on; go env GO111MODULE && go env GO111MODULE && go env GO111MODULE
on
on
on
```

Reply:
You are quite right! But I think this setting is not needed now, at all.

Updated the commit and the PR description, PTAL